app/assets/stylesheets/yourTrips.scss

```diff
@@ -0,0 +1,43 @@
+@import "variables";
+
+.your-trips-container {
+  text-align: center;
+  padding: 3% 0;
+  color: $blue-purple;
+
+  h1.yt-header-title {
+    font-family: $header;
+  }
+
+  .yt-card {
```

Comment:
I like the custom prefix of classes here! keep this up for other generic classes because classes like `.row` are too generic and likely to cause unintended overriding @Shazeen15 @KevinBurgin @Vivean28